infra/feast-operator/internal/controller/auth/auth_types.go

```diff
@@ -0,0 +1,30 @@
+package auth
+
+import (
+	feastdevv1alpha1 "github.com/feast-dev/feast/infra/feast-operator/api/v1alpha1"
+	"github.com/feast-dev/feast/infra/feast-operator/internal/controller/handler"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// FeastAuth is an interface for configuring feast authorization
+type FeastAuth struct {
+	Handler handler.FeastHandler
+}
+
+var (
+	nameLabelKey = feastdevv1alpha1.GroupVersion.Group + "/name"
```

Comment:
i'd like to only have this set once between `auth` and `services` package so we can ensure the labels stay in sync. any ideas on a location where both could import from?

Reply:
Changed this to reuse `services.NameLabelKey` instead. I'm not 100% convinced of that, but the `auth` package is also using other APIs and resources from the `services` package:
```
services.GetFeastName
services.GetFeastServiceName
services.NameLabelKey
```

We can move them in a common package or under `controller`, or just accept that `auth` depends on `services` and try to avoid cyclic dependencies.